Description: Add SolusOS credit (i.e. me)
 Added myself to the credits
 .
 nautilus (1:3.4.2-1.4solusos1) two; urgency=low
 .
   * Give thyself credit.
Author: Ikey Doherty <ikey@solusos.com>

---
Reviewed-By: Ikey Doherty <ikey@solusos.com>
Last-Update: <2012-07-07>

--- nautilus-3.4.2.orig/src/nautilus-window-menus.c
+++ nautilus-3.4.2/src/nautilus-window-menus.c
@@ -327,6 +327,7 @@ action_about_nautilus_callback (GtkActio
 		"Gene Z. Ragan",
 		"George Lebl",
 		"Ian McKellar",
+		"Ikey Doherty <ikey@solusos.com>\n",
 		"J Shane Culpepper",
 		"James Willcox",
 		"Jan Arne Petersen",
